app/src/main/java/org/ole/planet/myplanet/ui/team/TeamPagerAdapter.kt

```diff
@@ -26,22 +26,16 @@ class TeamPagerAdapter(fm: FragmentActivity, team: RealmMyTeam?, private val isI
     private val isEnterprise: Boolean = TextUtils.equals(team?.type, "enterprise")
 
     init {
+        list.add(MainApplication.context.getString(R.string.chat))
```

Comment:
moving this outside the `if (isInMyTeam || team?.isPublic == true) { ` condition means it will always be shown when it should `only` be shown if learner is in the team which they are a member or if the team is public so return it inside the condition